package.json

```diff
@@ -20,21 +20,30 @@
     "type": "git",
     "url": "git://github.com/brianloveswords/buffer-crc32.git"
   },
-  "main": "index.js",
   "scripts": {
-    "test": "tap tests/*.test.js --reporter classic"
+    "test": "tap tests/*.test.js --reporter classic",
+    "build": "unbuild",
```

Comment:
index.js was moved to src because that's where unbuild assumes files are, this allows to use a zero-configuration setup. Alternatively I could add explicit configuration